internal/pkg/crypto/io.go

```diff
@@ -189,13 +188,17 @@ func (w *Writer) writeBlock(b []byte) error {
 		return w.err
 	}
 
-	w.writer.Write(iv)
```

Comment:
linter complained: Error return value of `a.monitor.Cleanup` is not checked (errcheck)